config/config.go

```diff
@@ -34,6 +34,7 @@ type MixConfig struct {
 	Swupd   swupdConf
 	Server  serverConf
 	Mixer   mixerConf
+	Os      osConf
```

Comment:
Sure. I can change it to more specific name related to OsRelease. I will change the section name in the builder.conf as well. Use [OsRelease] instead of [Os].